apps/browser-extension-wallet/src/features/delegation/components/MultiDelegationStakingPopup.tsx

```diff
@@ -80,7 +84,9 @@ export const MultiDelegationStakingPopup = (): JSX.Element => {
     isSharedWallet: state.isSharedWallet
   }));
   const wallet = useCurrentWallet();
-  const { sharedWalletKey, coSigners } = useSharedWalletData(wallet);
+  const wallets = useObservable(walletRepository.wallets$);
+
+  const parentWalletCIP1854Account = getParentWalletCIP1854Account({ wallets, activeWallet: wallet });
```

Comment:
Naming it like this is more descriptive
```suggestion
  const parentMultiSigAccount = getParentMultiSigAccount({ wallets, activeWallet: wallet });
```

Reply:
done here bdd149b0772a065a5ed16957af37393fe2dc1fb7